using_images/s2i_images/perl.adoc

```diff
@@ -103,8 +103,8 @@ recommended to turn this on in your production environment.
 Hot deployment allows you to quickly make and deploy changes to your application
 without having to generate a new S2I build. To enable hot deployment in this
 image, you must set the `*PERL_APACHE2_RELOAD*` environment variable to *true*.
-You can use
-link:../../dev_guide/environment_variables.html#set-environment-variables[the
+For example, see link:../../dev_guide/new_app.html#specifying-environment-variables[the `oc new-app` command].
+You can use link:../../dev_guide/environment_variables.html#set-environment-variables[the
 `oc env` command] to update environment variables of existing objects.
```

Comment:
Unfortunately, when adding a link to words marked up with backticks, they don't look any different, so you can't tell it's a link unless you hover. Was trying to work around this by including a plaintext word or two around it, so it's more obvious there's a link. But I agree it's lame...